Frontend/src/app/app.component.html

```diff
@@ -1,2 +1,7 @@
+<app-header></app-header>
+<app-nav-user-search></app-nav-user-search>
```

Comment:
When you add this in all pages we have same components, and not for all it is appropriate (admin pages have own sidebar and etc). So you should put it for that components where it should be
![image](https://user-images.githubusercontent.com/75213424/154852555-bc816c3d-2c7a-4f30-9fd2-10f2eead9752.png)